bugsnag-plugin-android-anr/src/main/java/com/bugsnag/android/AnrPlugin.kt

```diff
@@ -103,6 +103,12 @@ internal class AnrPlugin : Plugin {
                 err.stacktrace.addAll(0, nativeTrace.map { Stackframe(it) })
             }
 
+            if (nativeTrace != null) {
+                for (error in event.errors) {
+                    error.type = ErrorType.C
+                }
+            }
+
```

Comment:
I don't think we want to do this - the frame type will (eventually) be set.

Reply:
Removed this override of frame type.